playbooks/roles/filebeat/templates/filebeat.yml.j2

```diff
@@ -0,0 +1,43 @@
+filebeat:
+  # List of prospectors to fetch data.
+  prospectors:
+    # Each - is a prospector. Below are the prospector specific configurations
+    -
+      paths:
+        - /var/log/syslog
+        - "/var/log/mail.*"
+        - "/var/log/kern.*"
+        - "/var/log/news.*"
+        - "/var/log/*.emerg"
+        - "{{ elasticsearch_log_dir }}/*"
+        - "{{ nginx_log_dir }}/*.log"
+        - "{{ nginx_log_dir }}/*.err"
+        - "{{ COMMON_LOG_DIR }}/tracking/*.log"
+        - "{{ COMMON_LOG_DIR }}/cms/edx.log"
+        - "/edx/app/edxapp/edx-platform/logs/util1/xqueue/edx.log"
+        - "{{ COMMON_LOG_DIR }}/devpi/supervisor/*"
+        - "/edx/app/edxapp/edx-platform/logs/util1/supervisor/*"
+        - "/edx/app/edxapp/edx-platform/logs/util1/notifier/edx.log"
+        - "/edx/app/edxapp/edx-platform/logs/util1/certs/edx.log"
+        - "/edx/app/edxapp/edx-platform/logs/app1/tracking/tracking.log"
+        - "/edx/app/edxapp/edx-platform/logs/app1/supervisor/*.log"
+        - "/edx/app/edxapp/edx-platform/logs/app1/nginx/*.log"
+        - "/edx/app/edxapp/edx-platform/logs/app1/lms/edx.log"
+        - "/edx/app/edxapp/edx-platform/logs/app1/cms/edx.log"
+        - "{{ COMMON_LOG_DIR }}/ecommerce/*"
+        - "{{ COMMON_LOG_DIR }}/ecommerce_worker/*"
+        - "{{ COMMON_LOG_DIR }}/lms/edx.log"
+        - "{{ logstash_log_dir }}/*.log"
+        - "{{ COMMON_LOG_DIR }}/mongo/*"
+        - "{{ COMMON_LOG_DIR }}/notifier/edx.log"
+        - "{{ COMMON_LOG_DIR }}/rabbitmq/*"
+        - "{{ COMMON_LOG_DIR }}/supervisor/*"
+
+      input_type: log
+
+output:
+
+  ### Logstash as output
+  logstash:
+    # The Logstash hosts
```

Comment:
This is what I would jokingly call "redundant comment redundancy."

In cases like this, if the comment isn't really adding anything useful to the conversation, then let's exclude it. From the context, here, a `hosts` key inside the `logstash` dictionary, it's safe to assume that we know this is `The Logstash hosts`.

You'll see this a lot like:
```
# This deletes the user record.
user.delete()
```

Documentation is _generally_ helpful, but here, it's a distraction at best. Worse, it's one more thing that needs to stay in sync w/ the surrounding code. We can get the best of all worlds by writing code that self-documents (which you already have here).